### PR TITLE
Added the option to create peers with other chrony servers.

### PR DESCRIPTION
**Pull request by JuddTracy-DAS**
I added the option to have chrony servers peer with other servers.  The change is minimal and essentially a copy and paste of the `chrony_ntp_servers` section in the template.

### CODE REVIEW

**Comment by Frzk** [comment]
Hi,

Thanks a lot for your time, help and interest.

It sounds good to me and I'll merge this. I just have a minor remark: as stated in [chrony.conf documentation](https://chrony.tuxfamily.org/doc/4.0/chrony.conf.html), the `peer` directive doesn't support the `iburst` option.
Would it be possible for you to remove this option from the example you added in `README.md` ?

Thanks :-)

File: README.md [hunk 1]
```diff
@@ -44,6 +45,10 @@ chrony_ntp_servers:
   - 1.rhel.pool.ntp.org iburst maxpoll 10
   - 2.rhel.pool.ntp.org iburst maxpoll 10
   - 3.rhel.pool.ntp.org iburst maxpoll 10
+chrony_ntp_peers:
+  - ntp00.example.com iburst maxpoll 10
+  - ntp01.example.com iburst maxpoll 10
+  - ntp02.example.com iburst maxpoll 10
```
Comment (Frzk) [inline README.md:51]:
```suggestion
  - ntp00.example.com maxpoll 10
  - ntp01.example.com maxpoll 10
  - ntp02.example.com maxpoll 10
```

File: templates/chrony.conf.j2 [hunk 2]
```diff
@@ -16,6 +16,13 @@ pool {{ pool }}
 {% endfor %}
 {% endif %}
 
+{% if chrony_ntp_peers|length > 0 %}
+# List of NTP Peers to use
```
Comment (Frzk) [inline templates/chrony.conf.j2:20]:
```suggestion
# List of NTP peers to use
```

**Review by Frzk — CHANGES_REQUESTED** [review]

**Comment by JuddTracy-DAS** [comment]
Thanks for approving this so quickly.

**Pull request merged**